### ISSUE TITLE
[feature already present-in part] look , theres is a simple way to test your own browser with your settings see the link https://privacytests.org/me.html

### ISSUE THREAD

**Issue by y1111** [privacytests/privacytests.org#198]
https://privacytests.org/me.html

from this post here https://www.reddit.com/r/browsers/comments/1adnm2n/i_run_the_httpsprivacytestsorg_in_my_browsers/
theres tests made in "tested floorp , chrome , vivaldi , thorium , edge . with ublock and not"
and the link were the test was found https://www.reddit.com/r/IAmA/comments/1abkmj3/comment/kjr9aww/?utm_source=share&utm_medium=web2x&context=3

i leave it here for other people to run their tests and make another issue with tests in android i made

**Comment by JannisBush** [comment]
@arthuredelstein 
I just found this link today. Maybe there should be a `test yourself` button somewhere on the page? However to make it really useful #187 needs to be completed first.

**Comment by burdges** [comment]
It doesn't work anymore:  https://github.com/privacytests/privacytests.org/issues/123 https://github.com/privacytests/privacytests.org/issues/249
